docs/guide/core-concepts.rst

```diff
@@ -1673,18 +1673,114 @@ things. Cols run east to west whereas Rows run north to south. This then
+and munging (metadata for each individual GeoTiff will need to be collected
+as opposed to the monolithic metadata of Avro layers, which is read and
+handled once for the entire layer).
+
+The `COG specification <http://www.cogeo.org/>`_ (which is not a 100%
+complete as of the writing of this documentation) defines required
+tags and means of access (a server accepting GET range requests). These
+required features are necessary to even support remotely reading
+subsets of the overall image data from some remote Tiff.
+
+COG requirements:
+- Tiled storage of image data
+- Overviews at different levels of resolution
+- Infrastructure capable of handling GET Range requests
+
+.. code-block:: scala
+
+    // Constructing a COG from a non-COG tiff
+    val nonCog = SinglebandGeoTiff(path = file:///path/to/my/tiff.tif)
+    val almostCog = nonCog.withStorageMethod(Tiled)
+
+    // most likely either NearestNeighbor or BilinearInterpolation; depends on context
+    val resampleMethod: ResampleMethod = ???
+    val fullCog = almostCog.withOverviews(resampleMethod)
+
+> A note on sidecar files
+> The spec seems to indicate that overviews be part of the GeoTiff itself to
+> count as a COG. In practice, things are messier than that. Content providers
+> aren't always going to want to rewrite their tiffs to stuff generated
+> overviews into them. The practical upshot of this is that separate overview
+> files should be supported (GDAL will actually inspect some canonical relative
+> paths within the directory of the Tiff being read).
+
+.. code-block:: scala
+
+    // Constructing a COG with sidecar overviews
+    val mainTile = SinglebandGeoTiff(path = file:///path/to/my/file.tif)
+    val overview1 = SinglebandGeoTiff(path = file:///path/to/my/file.tif.ovr1)
+    val overview2 = SinglebandGeoTiff(path = file:///path/to/my/file.tif.ovr2)
+    val tileWithOverview = mainTile.withOverviews(List(overview1, overview2))
+
+Structured vs Unstructured COGs
+-------------------------------
+
+Historically, Geotrellis layers have been backed by specially encoded Avro
+layers which are were designed to maximize the performance of distributed
+reading and querying. With the advent of the COG and the development of
+tooling to support this subset of the GeoTiff spec, however, the advantages
+of depending upon a bespoke raster format are less obvious than they once were.
+Avro support is likely to continue, but support for applications backed by
+COGs are a priority for continued GeoTrellis development.
+
+To this end, GeoTrellis is introducing the notion of a 'structured' COG layer.
+Structured COG layers are actually a collection of COGs tiled out in a
+consistent manner and described through common (GeoTrellis-specific) metadata
+which is designed to enhance query performance for larger layers by allowing
+GeoTrellis programs to infer information about underlying, individual COG files
+without having to read multiple of them.
+
+Structured COG metadata:
```

Comment:
The formatting for this also appears to be incorrect and doesn't produce the correct, visual result.